src/reanimated2/platform-specific/webTransitions.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  WEB_ANIMATIONS_ID,
+  generateRandomKeyframeName,
+  setElementAnimation,
+} from './webAnimations';
+
+import type { AnimationConfig } from './webAnimations';
+
+export enum TransitionType {
```

Comment:
I'm not sure, but to mirror native usage `Linear` should probably have `Layout` alias here, cc @tomekzaw.

Reply:
@piaskowyk, what do you think?